examples/fruitflies.py

```diff
@@ -33,9 +33,13 @@
 def make_png(filename):
     tree.pydot__tree_to_png( parser.parse(sentence), filename)
 
+def make_dot(filename):
+    tree.pydot__tree_to_dot( parser.parse(sentence), filename)
+
 if __name__ == '__main__':
     print(parser.parse(sentence).pretty())
     # make_png(sys.argv[1])
+    # make_dot(sys.argv[1])
```

Comment:
Why two comments?

Reply:
It was to expose the use of dot output, but now it seems that defining `make_do` is enougth.